src/commands/snapshot.ts

```diff
@@ -0,0 +1,105 @@
+import {flags} from '@oclif/command'
+import Constants from '../services/constants'
+import {StaticSnapshotOptions} from '../services/static-snapshot-options'
+import StaticSnapshotService from '../services/static-snapshot-service'
+import PercyCommand from './percy-command'
+
+export default class Snapshot extends PercyCommand {
+  static description = 'Snapshot a directory of static site assets'
+  static hidden = true
+
+  static args = [{
+    name: 'staticAssets',
+    description: 'The path to the compiled static asset directory.',
+    required: true,
+  }]
+
+  static examples = [
+    '$ percy snapshot _mySite/',
+    '$ percy snapshot _mySite/ --baseUrl "blog/"',
+    '$ percy snapshot _mySite/ --widths "320,780,1280"',
+    '$ percy snapshot _mySite/ --ignore-folders "Tmp,_secrets,node_modules"',
+  ]
+
+  static flags = {
+    'snapshot-capture-regex': flags.string({
+      char: 'c',
+      description: 'Regular expression for matching the files to snapshot. Defaults to: "\.(html|htm)$"',
+      default: '\.(html|htm)$',
+    }),
+    'ignore-folders': flags.string({
+      char: 'i',
+      description: 'Comma-seperated string of folders to ignore. Ex: Tmp,_secrets,node_modules',
+      default: '',
+    }),
+    'widths': flags.string({
+      char: 'w',
+      description: 'Comma-separated string of rendering widths for snapshots. Ex: 320,1280',
+      default: '1280',
+    }),
+    'baseUrl': flags.string({
+      char: 'b',
+      description: 'The path that the site will be deployed to on a production server. \
+      Use this if your site will be hosted at a non-root url.',
+      default: '/',
+    }),
+    // from exec command. needed to start the agent service.
+    'network-idle-timeout': flags.integer({
+      char: 't',
+      default: 50,
```

Comment:
Wonder if this should be moved to the constants?

Reply:
Yup, I agree. I was just thinking about that yesterday

Reply:
That would be great. @maprules1000 can you draw inspiration from the `Constants.PORT` usage below? Be sure to apply the same change to the other commands.